src/DDTrace/Transport/Http.php

```diff
@@ -96,14 +99,23 @@ public function getConfig()
 
     private function sendRequest($url, array $headers, $body)
     {
+        $bodySize = strlen($body);
```

Comment:
Question: how is this working with msgpack binary protocol we are about to merge?

Reply:
Very good thing to check! The `strlen()` [returns the number of bytes in the string](https://www.php.net/strlen), not the number of characters like `mb_string()` so it is safe to use in this context since we're measuring total payload size vs number of characters in the payload. :)